README.md

```diff
@@ -52,6 +52,7 @@ gcc. The project uses `<stdint.h>` which is technically a C99 file and not
 available in Microsoft Visual Studio. The easiest solution here is to use the
 msinttypes version of stdint.h from <https://code.google.com/p/msinttypes/>.
 The direct link for the lazy people is:
+>>> The below Link is not working.
```

Comment:
Maybe you can refer to https://raw.githubusercontent.com/chemeris/msinttypes/f9e7c5758ed9e3b9f4b2394de1881c704dd79de0/stdint.h